### PR TITLE
website: minor edits to "Installing" pages and "Go SDK" page

### PR DESCRIPTION
**Pull request by NoahMundahl**

### CODE REVIEW

File: website/content/docs/installing/systemd.mdx [hunk 1]
```diff
@@ -45,8 +43,8 @@ WantedBy=multi-user.target
 
 ## Systemd All-in-One Installation Script
 
-Here's a simple install script that creates the boundary group and user, installs the
-systemd unit file and enables it at startup:
+Here's a simple install script that creates the Boundary group and user, installs the
```
Comment (jefferai) [inline website/content/docs/installing/systemd.mdx:46]:
```suggestion
Here's a simple install script that creates the `boundary` group and user, installs the
```

File: website/content/docs/installing/high-availability.mdx [hunk 2]
```diff
@@ -40,15 +35,15 @@ Boundary requires an external [Postgres](https://www.postgresql.org/) and [KMS](
...
 For general configuration, we recommend the following:
 
 - HTTPS listener with valid TLS certificate for the domain it's serving or TLS passthrough
-- Health check port should use :9200 with TCP protocol
+- Health checks should use TCP port 9200
 
 ### Controller Configuration
 
-When running Boundary controller as a service we recommend storing the file at `/etc/boundary-controller.hcl`. A `boundary` user and group should exist to manage this configuration file and to further restrict who can read and modify it.
+When running Boundary controller as a service, we recommend storing the file at `/etc/boundary-controller.hcl`. A `boundary` user and group should exist to manage this configuration file and to further restrict who can read and modify it.
```
Comment (jefferai) [inline website/content/docs/installing/high-availability.mdx:47]:
```suggestion
When running Boundary controller as a service we recommend storing the file at `/etc/boundary-controller.hcl`. A `boundary` user and group should exist to manage this configuration file and to further restrict who can read and modify it.
```

File: website/content/docs/api-clients/go-sdk.mdx [hunk 3]
```diff
@@ -37,21 +37,19 @@ method, login name, and password are pre-configured.
 First, we need to create a client from the Boundary API and set the address to reach Boundary:
 
 ```go
-import "github.com/hashicorp/boundary/api"
-
 // The default address points to the default dev mode address
 client, err := api.NewClient(nil)
 if err != nil {
-  return err
+  fmt.Printf("ERROR: %v\n", err.Error())
```
Comment (jefferai) [inline website/content/docs/api-clients/go-sdk.mdx:43]:
```suggestion
  return err
```

File: website/content/docs/api-clients/go-sdk.mdx [hunk 4]
```diff
@@ -50,8 +48,8 @@ The [authenticate
 method](https://github.com/hashicorp/boundary/blob/main/api/authmethods/authenticate.go)
 uses a basic `map[string]interface{}` to pass credential information, in order
 to eventually support multiple auth method types. For this example, we assume
-you're using the password auth method and so we're going to tailor a credentials
-object to pass this data as:
+you're using the password auth method, and so we create a credentials
```
Comment (jefferai) [inline website/content/docs/api-clients/go-sdk.mdx:51]:
```suggestion
you're using the password auth method, and so we create an attributes
```

File: website/content/docs/api-clients/go-sdk.mdx [hunk 5]
```diff
@@ -89,46 +85,53 @@ Note the ID in the output above, we're going to use that in the next step.
 We can use the credentials object we created to execute `Authenticate()` on this client:
 
 ```go
-at, err := am.Authenticate(context.Background(), "ampw_1234567890", credentials)
+authenticationResult, err := amClient.Authenticate(context.Background(), "ampw_1234567890", "login", credentials)
 if err != nil {
-  return err
+  fmt.Printf("ERROR: %v\n", err.Error())
```
Comment (jefferai) [inline website/content/docs/api-clients/go-sdk.mdx:90]:
```suggestion
  return err
```

File: website/content/docs/api-clients/go-sdk.mdx [hunk 6]
```diff
@@ -99,36 +95,43 @@ Lastly, let's update the original client with the token we got from the `Authent
...
 ```go
 // pass this client to any other resource specific API resources
-client.SetToken(at.Item.Token)
+client.SetToken(fmt.Sprint(authenticationResult.Attributes["token"]))
 ```
 
 Putting this all together:
 
 ```go
+package main
```
Comment (jefferai) [inline website/content/docs/api-clients/go-sdk.mdx:104]: I don't think this should be structured as a complete program -- it's not. It doesn't do anything useful, and it's not fully structured -- `main` doesn't even return. The example is meant as integration advice within a program.

Comment (NoahMundahl) [reply website/content/docs/api-clients/go-sdk.mdx:104]: Makes sense.  I will update to reflect that goal.

**Comment by jefferai** [comment]
Thanks!

**Pull request merged**